test/README.md

```diff
@@ -0,0 +1,28 @@
+# osbuild-composer testing information
+
+
+## Integration testing
+
+This will consume the osbuild-composer API surface via the `composer-cli`
+command line interface. Implementation is under `cmd/osbuild-tests/`.
+
+The easiest way to get started with integration testing from a git
+checkout is:
+
+* `make rpm` to build the software under test
+* `dnf install output/x86_64/golang-github-osbuild-composer-*.rpm` - this will
+  install both osbuild-composer, its -debuginfo, -debugsource and -tests packages
+* `systemctl start osbuild-composer`
+* `/usr/libexec/tests/osbuild-composer/osbuild-tests` to execute the test suite
```

Comment:
We advise to use a fresh system installation to running these, would you like to mention it?

Otherwise I like more documentation :-)